beacon-chain/blockchain/process_block.go

```diff
@@ -501,6 +501,15 @@ func (s *Service) handleEpochBoundary(ctx context.Context, postState state.Beaco
 		if err := helpers.UpdateProposerIndicesInCache(ctx, copied); err != nil {
 			return err
 		}
+
+		if err := copied.SetSlot(copied.Slot() + params.BeaconConfig().SlotsPerEpoch); err != nil {
+			return err
+		}
+		// This is a no-op for caching proposer indices for epoch + 1.
+		_, err = helpers.BeaconProposerIndex(ctx, copied)
+		if err != nil {
+			log.Warn(ctx, "failed to get proposer index to cache next epoch proposers", "err", err)
```

Comment:
```suggestion
			log.Warn(ctx, "Failed to cache next epoch proposers", "err", err)
```
Removing the part about proposer index because we don't care about it.